rero_ils/modules/serializers/base.py

```diff
@@ -205,3 +205,36 @@ def get_resource(self, loader, pid):
                 resource = resource.to_dict()
             self._resources.setdefault(cls_key, {})[pid] = resource
         return self._resources[cls_key][pid]
+
+
+class StreamedSerializerMixin:
```

Comment:
Name of mixin class seems little bit strange in my sens.
Maybe we can rewrite his to `StreamSerializerMixin` and do something like this:

```python
class StreamSerializerMixin(SerializerMixinInterface, 
                            PostprocessorMixinInterface):

    chunk_size = 1000
    
    @classmethod
    def get_chunks(cls, results):
        ...
    
    def serialize_search(self, pid_fetcher, search_result, links=None,
                         item_links_factory=None, **kwargs):
        return stream_with_context(
            self.postprocess_serialize_search(search_result)
        )
```